{{cookiecutter.project_shortname}}/Dockerfile.base

```diff
@@ -0,0 +1,7 @@
+{% include 'misc/header.py' %}
+FROM inveniosoftware/centos7-python:36
```

Comment:
See the image naming/versioning comments at https://github.com/inveniosoftware/docker-invenio/pull/6#discussion_r212348438

`centos7-python:36` is not friendly towards image upgrades... How would the version string look like for Python upgrades (3.7.1 -> 3.7.2) or for Invenio prerequisite upgrades (3.0.0 -> 3.4.2)?